hack/local-up-cluster.sh

```diff
@@ -750,7 +750,7 @@ function start_kubelet {
 
     # shellcheck disable=SC2206
     all_kubelet_flags=(
-      "${priv_arg}"
+      ${priv_arg}
```

Comment:
is this a single flag when set?

Reply:
yes it is https://cs.k8s.io/?q=priv_arg&i=nope&files=&repos=kubernetes/kubernetes

Reply:
`${priv_arg:+"$priv_arg"}` will avoid splitting issues and _also_ not become an empty string

Reply:
@caesarxuchao let's do `${priv_arg:+"$priv_arg"}`, this will expand to a single flag (shell word / array entry) if set to a non-empty string and otherwise not expand to anything.